includes/definitions/discovery/linux.yaml

```diff
@@ -0,0 +1,142 @@
+                    descr: 'BBU'
+                -
+                    oid: physicalDriveTable
+                    value: pdTemperature
+                    num_oid: '.1.3.6.1.4.1.3582.4.1.4.2.1.2.1.36.{{ $index }}'
+                    index: 'slot.{{ $index }}'
+                    group: 'RAID Controller {{ $adpID-PDT }} Enclosure {{ enclDeviceId }}'
+                    descr:  '/c{{ $adpID-PDT }}/e{{ $enclDeviceId }}/s{{ $slotNumber }}'
+        count:
+            data:
+                -
+                    oid: adapterPropertiesTable
+                    value: vdPresentCount
+                    num_oid: '.1.3.6.1.4.1.3582.4.1.4.1.2.1.18.{{ $index }}'
+                    index: 'vdsum.{{ $index }}'
+                    group: 'RAID Controller {{ $adapterID-APT }}'
+                    descr: 'Virtual disks'
+                -
+                    oid: adapterPropertiesTable
+                    value: pdDiskPredFailureCount
+                    num_oid: '.1.3.6.1.4.1.3582.4.1.4.1.2.1.19.{{ $index }}'
+                    index: 'vdfail.{{ $index }}'
+                    group: 'RAID Controller {{ $adapterID-APT }}'
+                    warn_limit: 1
+                    descr: 'Virtual disk failure'
+                -
+                    oid: adapterPropertiesTable
+                    value: pdDiskPredFailureCount
+                    num_oid: '.1.3.6.1.4.1.3582.4.1.4.1.2.1.23.{{ $index }}'
+                    index: 'criticaldisks.{{ $index }}'
+                    group: 'RAID Controller {{ $adapterID-APT }}'
+                    warn_limit: 1
+                    descr: 'Critical disks'
+                -
+                    oid: adapterPropertiesTable
+                    value: pdDiskFailedCount
+                    num_oid: '.1.3.6.1.4.1.3582.4.1.4.1.2.1.24.{{ $index }}'
+                    index: 'faileddisks.{{ $index }}'
+                    group: 'RAID Controller {{ $adapterID-APT }}'
+                    warn_limit: 1
+                    descr: 'Failed disks'
+        state:
+            data:
+                -
+                    oid: adapterPropertiesTable
+                    value: alarmState
+                    num_oid: '.1.3.6.1.4.1.3582.4.1.4.1.2.1.4.{{ $index }}'
+                    group: 'RAID Controller {{ $adapterID-APT }}'
+                    descr: 'Alarm'
+                    state_name: alarmState
+                    states:
+                        - { descr: disabled , graph: 0, value: 0, generic: 1 }
+                        - { descr: enabled  , graph: 0, value: 1, generic: 0 }
+                -
+                    oid: enclosurePowerSupplyTable
+                    value: powerSupplyStatus
+                    num_oid: '.1.3.6.1.4.1.3582.4.1.5.5.1.3.{{ $index }}'
+                    group: 'Enclosure {{ $enclosureId-EPST }}'
+                    descr: 'Powersupply {{ $powerSupplyID }}'
+                    state_name: powerSupplyStatus
+                    states:
```

Comment:
Does someone knows, how to use the same states multiple times?

Reply:
Hi @mpikzink 
To my knowledge, it is not (yet) possible in the YAML syntax. But all patches are welcomed 👍

Reply:
Thanks @PipoCanaja I put it on the to-do list :-)